fedimint-server/src/config/mod.rs

```diff
@@ -208,6 +208,10 @@ pub struct ConfigGenSettings {
     pub api_url: Option<SafeUrl>,
     /// Enable iroh for networking
     pub enable_iroh: bool,
+    /// Optional URL of the Iroh DNS server
+    pub iroh_dns: Option<SafeUrl>,
+    /// Optional URLs of the Iroh relays
```

Comment:
[nit] Plural is a bit confusing since we can only specify a single URL

Reply:
I think the intention was the ability to register multiple relays (then Iroh selects the one that is closest, and this also gives everything some redundancy), so I just implemented it now.